src/adapters/MeetingsJSONAdapter.js

```diff
@@ -101,13 +101,21 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       )
     );
 
+    const getMeetingWithRemoteMedia$ = getMeeting$.pipe(
```

Comment:
I think this is better to be done as part of `getMeetingWithLocalMedia$`'s pipe. This way, we would get a meeting object that has all the streams all at once